packages/typography/stories/14-UnorderedList.stories.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { UnorderedList } from '@zendeskgarden/react-typography';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Typography/UnorderedList',
+  subcomponents: {
+    UnorderedList,
+    'UnorderedList.Item': UnorderedList.Item
+  }
+} as Meta;
+
+export const Default: Story = ({ size, type }) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <UnorderedList size={size} type={type} style={{ display: 'inline-block' }}>
+          <UnorderedList.Item>Plant</UnorderedList.Item>
+          <UnorderedList.Item>Water</UnorderedList.Item>
```

Comment:
The indentation problem appears on the first two items. Again, not sure what's going on.